include/micm/solver/rosenbrock.hpp

```diff
@@ -353,7 +369,7 @@ namespace micm
                 K[stage][idx] += HC * K[j][idx];
               }
             }
-            K[stage] = lin_solve(K[stage], ode_jacobian);
+            lin_solve(K[stage], K[stage]); // This probably won't work! FIXME
```

Comment:
is this fixed?

Reply:
not yet, this is one reason I commented out this block until we get to #112